server/netty-server/sync/src/main/scala/sttp/tapir/server/netty/sync/internal/reactivestreams/OxProcessor.scala

```diff
@@ -63,11 +71,15 @@ private[sync] class OxProcessor[A, B](
       val channelSubscription = new ChannelSubscription(wrappedSubscriber, outgoingResponses)
       subscriber.onSubscribe(channelSubscription)
       channelSubscription.runBlocking() // run the main loop which reads from the channel if there's demand
-    } { error =>
+    }(pipelineFork) { error =>
       wrappedSubscriber.onError(error)
       onError(error)
     }
 
+  private def cancelPipelineFork(): Unit =
+    try { pipelineFork.get(pipelineCancelationTimeoutMs, TimeUnit.MILLISECONDS).cancelNow() }
+    catch case NonFatal(_) => ()
```

Comment:
do we really want to ignore this exception? :)

Reply:
We shouldn't throw from here, but we could at least log it, right?

Reply:
What happens if we throw? I guess it's a bug if the future isn't completed with the CancellableFork?

Reply:
Throwing here may be against reactive streams specs and can lead to unpredicted behavior. This can cause Netty to get into an unhandled exception state without closing resources properly.

Reply:
ok, so logging it is :)